src/strategies/multichain/index.ts

```diff
@@ -62,6 +63,43 @@ async function getChainBlocks(
   return chainBlocks;
 }
 
+async function getChainBlocksFromApi(
+  snapshot,
+  provider,
+  options
+): Promise<any> {
+  const block = await provider.getBlock(snapshot);
+  const timestamp = block.timestamp;
+
+  //Should receive the timestamp as query param and return the block height by chain
+  const apiUrl = `${options.blockApi}?timestamp=${timestamp}`;
+  const resp = await fetch(apiUrl, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    }
+  }).then(r => {
+    return r.json()
+  });
```

Comment:
```suggestion
  }).then(r => r.json());
```

![magic](https://user-images.githubusercontent.com/6792578/147301015-83f705d0-dea1-411a-a8f0-aad4f2236d3b.gif)